posthog/migrations/0368_user_theme_mode.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 3.2.19 on 2023-11-29 19:14
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("posthog", "0367_job_inputs"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="user",
+            name="theme_mode",
+            field=models.CharField(
+                blank=True, choices=[("light", "Light"), ("dark", "Dark")], max_length=20, null=True
```

Comment:
Is the default here `null`? That would mean that 3000 will match the users system preferences when it launches for every user. Is it common to change a tool to dark mode for folks when it becomes available or should we start everyone on light?

Reply:
The default is null, yeah. I do think we should match the system theme by default, especially since light mode is also different now – so either way the color scheme is much different for the user